### PR TITLE
Autosize timesheet notes textarea only on focus to avoid incorrect height and scroll issue

### LINKED ISSUE #689: Timesheet entry description field is not scrollable or expandable for edit
### Description

When creating, If we add a longer description to the timesheet entry, we couldn't be able to scroll through the timesheet entry description while editing.

### Environment

    Linux / MacOS / Android / IOS / Windows
    Mozilla / Chrome / Safari

### What is the expected behaviour?

When we try to edit the timesheet entry, the description field should be scrollable.

### What is the current behaviour?

we couldn't be able to scroll through the timesheet entry description while editing.

### How to reproduce the issue?

1. Go to the time tracking page
2. Create a new timesheet entry with a longer description (use 150 words min)
3. Edit the timesheet entry.
4. Try to scroll the description.

### Screenshots or Screencast

https://www.loom.com/share/611221e8860742feaf0008eebc6f758c

### PR DESCRIPTION
**Pull request by aaj013**
## Summary
Closes #689

Ref: https://www.jacklmoore.com/autosize/#faq-hidden

## Preview
[miru-textarea-autosize-fix.webm](https://user-images.githubusercontent.com/5847582/193429373-ade0524c-a535-4bd0-8e5c-edf9f13f2353.webm)

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [x] I have manually tested all workflows
- [x] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

**Review by prasanthchaduvula — CHANGES_REQUESTED** [review]

File: app/javascript/src/components/time-tracking/AddEntry.tsx [hunk 1]
```diff
@@ -49,7 +49,10 @@ const AddEntry: React.FC<Iprops> = ({
 
   useEffect(() => {
     const textArea = document.querySelector("textarea");
-    autosize(textArea);
+    // Delay assigning Autosize until the user focuses the textarea to avoid incorrect height (https://www.jacklmoore.com/autosize/#faq-hidden)
+    textArea.addEventListener("focus", function (){
+      autosize(textArea);
+    });
```
[the remark this answers is not in the document]
Comment (aaj013) [reply app/javascript/src/components/time-tracking/AddEntry.tsx:55]: Sure. Will do

**Comment by supriya3105** [comment]
@prasanthchaduvula Can you look at this again. Seems there was another deployment after your review yesterday. 
@apoorv1316 Please test this and merge once approved by Prasanth.

**Review by prasanthchaduvula — APPROVED** [review]
LGTM

**Comment by aaj013** [comment]
> @aaj013 Please resolve the conflicts

Hi, I am out of town for one week and internet access is very difficult here. It would be great if someone can help with resolving conflicts. Actually, there were no conflicts when I pushed my last commit. Don't know how it happened!

**Review by apoorv1316 — APPROVED** [review]

**Pull request merged**